Resources/Prototypes/Entities/Objects/Weapons/Guns/Pistols/pistols.yml

```
@@ -140,7 +140,7 @@
   name: cobra
   parent: [ BaseWeaponPistol, BaseSyndicateContraband ]
   id: WeaponPistolCobra
-  description: A rugged, robust operator handgun with inbuilt silencer. Uses .25 caseless ammo.
+  description: Integrally suppressed pistol adopted by The Syndicate, it uses unconventional caseless cartridges. Favored by any agent who likes to keep things quiet. Feeds from .25 pistol magazines.
```

This description could describe things more clearly.

```suggestion
  description: Integrally suppressed pistol adopted by the Syndicate, chambered in .25 caseless. Favored by any agent who likes to keep things quiet and leave nothing behind.
```